Makefile

```diff
@@ -105,11 +105,20 @@ addcopy: ## Add missing license to files
 .PHONY: generate
 generate: gen-docs gen-mocks gen-api-commands ## Generate docs, mocks, code, api commands, all auto generated assets
 
+.PHONY: apply-overlay
+apply-overlay: ## Apply overlay on openapi spec
+	@echo "==> Applying overlay"
+	go run ./tools/cmd/apply-overlay --spec ./tools/internal/specs/spec.yaml --overlay ./tools/internal/specs/overlays/\*.yaml > ./tools/internal/specs/spec-overlay.yaml
```

Comment:
Could we rename this output spec to something else? `spec-overlay.yaml`, sounds like it's the overlay, not the result.